backend/src/north/north-oianalytics/north-oianalytics.ts

```diff
@@ -105,8 +107,8 @@ export default class NorthOIAnalytics extends NorthConnector<NorthOIAnalyticsSet
 
     if (!response.ok) {
       throw {
-        message: `Error ${response.status}: ${response.statusText}`,
-        retry: [401, 403, 404, 500, 502, 503, 504].includes(response.status)
+        message: `Error ${response.statusCode}: ${await response.body.text()}`,
+        retry: [401, 403, 404, 500, 502, 503, 504].includes(response.statusCode)
       };
```

Comment:
Let's build an OIBusError with the same logic than handleFile. Let's also use the same error code than North Rest for the retry.